open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-advanced.ipynb

```diff
@@ -11,14 +11,14 @@
    "cell_type": "markdown",
```

Comment:
<p>why 'Fill the missing pieces' is a title?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I supposed it would be more hierarchical to do so, but it seems to fail. </p><p>I will del titles like <code>Fill the missing pieces</code>later</p>